src/zero/module/aou.clj

```diff
@@ -1,23 +1,385 @@
+                                           :rex               "greetings!"})
+  (extract-and-validate-per-sample-inputs {:chip_well_barcode           "7991775143_R01C01",
+                                           :rex                         "greetings!"
+                                           :bead_pool_manifest_file     "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExome-12v1-1_A.bpm",
+                                           :analysis_version_number     1,
+                                           :extended_chip_manifest_file "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExome-12v1-1_A.1.3.extended.csv",
+                                           :red_idat_cloud_path         "gs://broad-gotc-test-storage/arrays/HumanExome-12v1-1_A/idats/7991775143_R01C01/7991775143_R01C01_Red.idat",
+                                           :zcall_thresholds_file       "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/IBDPRISM_EX.egt.thresholds.txt",
+                                           :reported_gender             "Female",
+                                           :cluster_file                "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExomev1_1_CEPH_A.egt",
+                                           :sample_lsid                 "broadinstitute.org:bsp.dev.sample:NOTREAL.NA12878",
+                                           :sample_alias                "NA12878",
+                                           :green_idat_cloud_path       "gs://broad-gotc-test-storage/arrays/HumanExome-12v1-1_A/idats/7991775143_R01C01/7991775143_R01C01_Grn.idat",
+                                           :params_file                 "gs://broad-gotc-test-storage/arrays/HumanExome-12v1-1_A/inputs/7991775143_R01C01/params.txt",
+                                           :gender_cluster_file         "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExomev1_1_gender.egt"}))
+
+(defn make-inputs
+  "Return inputs for AoU Arrays processing in ENVIRONMENT from PER-SAMPLE-INPUTS."
+  [environment per-sample-inputs]
+  (let [inputs (merge references/hg19-arrays-references
+                      #_per-sample
+                      (extract-and-validate-per-sample-inputs per-sample-inputs)
+                      chip-metadata
+                      fingerprinting
+                      genotype-concordance
+                      other-inputs
+                      (env-inputs environment))]
+    (util/prefix-keys inputs :Arrays)))
+
+(defn make-options
+  "Return options for aou arrays pipeline."
+  []
+  {; TODO: add :final_workflow_outputs_dir here
+   ; TODO: add :default_runtime_attributes {:maxRetries 3} here
+   :read_from_cache            true
+   :write_to_cache             true
+   :default_runtime_attributes {:zones "us-central1-a us-central1-b us-central1-c us-central1-f"}})
+
+(defn active-or-done-objects
```

Comment:
Does this get used anywhere at the moment?

Reply:
This is not being used, but I tend to keep it since we are very likely to use it in the future. WDYT?

Reply:
Ok, let's keep it in then!